src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String WHITE = "white";
+    private static final int RADIUS_DEFAULT = 10;
+    private static final int MAX_RANDOM = 25;
+    private static final int ONE = 1;
+    private Random random = new Random();
+    private int randomInt;
+
+    private int randomNumberForLegs() {
+        int numericFofLegs;
+        numericFofLegs = random.nextInt(MAX_RANDOM) + ONE;
+        return numericFofLegs;

```

Comment:
```suggestion
    private int randomNumberForLegs() {
        return random.nextInt(MAX_RANDOM) + ONE;
```

you do not need this method, you can immediately call random.nextInt(MAX_RANDOM)
But if you want to keep it, you don't need to use a variable here, just do (see code)